packages/immutable-rootfs/build.yaml

```diff
@@ -13,7 +13,7 @@ copy:
 - package: 
     category: "toolchain"
     name: "yip"
-    version: ">=0"
+    version: ">=0.9.22"
```

Comment:
This should go in the collection.yaml file of the package as it is a runtime deps, not really a build dep (as we don't run any `yip` command inside the `build.yaml` file

Reply:
woops, sorry, screwed it up due checking that raw image test issue, will fix it later after checking that one

Reply:
wait, isnt it necessary during build so it can be using during module-setup.sh? thats what it says on the comment no? what would be the difference here on build VS runtime?